weave

```diff
@@ -1480,11 +1480,12 @@ proxy_args() {
 
     if [ -z "$PROXY_TLS_ENABLED" -a -z "$PROXY_TLS_DETECTION_DISABLED" ] ; then
         if ! DOCKER_TLS_ARGS=$(util_op docker-tls-args) ; then
-            echo "Unable to auto-detect proxy TLS configuration; you must launch the proxy with" >&2
-            echo "'weave launch-proxy' and supply '--no-detect-tls' or TLS options." >&2
-            exit 1
+            echo -n "Warning: skipping auto-detection of TLS configuration;" >&2
+            echo -n " to enable TLS, you must launch the proxy with" >&2
+            echo "'weave launch-proxy' and supply TLS options." >&2
```

Comment:
"Warning: unable to detect proxy TLS configuration. To enable TLS, launch the proxy with 'weave launch-proxy' and supply TLS options. To suppress this warning, supply the `--no-detect-tls` option."